src/main/webapp/js/angularApp/controllers.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+/* global kanban, _, $, angular */
+
+kanban.controller('kanbanController', ['Issues', '$rootScope', '$scope', '$timeout', function(Issues, $rootScope, $scope, $timeout) {
+
+  //default request - sending a click to the TL Staff button
+  $timeout(function() {
+    angular.element('#tlstaffbutton').triggerHandler('click');
+  }, 0);
+  // what collection is being shown is stored in the root scope
+  // not used now but might come in handy later
+  //var issuesUrl = '/jirarequest?' + $rootScope.sourceCollection;
+  
+  var wipUrl='/jirarequest?wip';
+  
+  // use a factory as a promise to get WIP data
+  Issues.getWip(wipUrl).then(function(result) {
+    if (result.data.errors) {
+      //source error: do something
```

Comment:
Is source error: do something intentional? Or should there be error handling logic here?